toolkit/ar/src/main/java/com/arcgismaps/toolkit/ar/internal/ArSessionWrapper.kt

```diff
@@ -53,6 +53,7 @@ internal class ArSessionWrapper(
     override fun onDestroy(owner: LifecycleOwner) {
         withLock { session, _ ->
             this.session = null
+            session?.pause()
             session?.close()
```

Comment:
the official doc also says to do this on a background thread. Maybe we should launch this on `Dispatchers.Default` as well
```suggestion
            withContext(Dispatchers.Default) {
                session?.close()
            }
```

Edit: my bad, we are not on a coroutine context, so this will need further thinking...